src/olympia/devhub/templatetags/jinja_helpers.py

```diff
@@ -88,10 +87,12 @@ def summarize_validation(validation):
 
 @library.global_function
 def pending_activity_log_count_for_developer(version):
-    alog = ActivityLog.objects.for_versions(version).filter(
-        action__in=amo.LOG_REVIEW_QUEUE_DEVELOPER
-    )
-    return filter_queryset_to_pending_replies(alog).count()
+    # unread_count is an annotation set by version_list() view to do this once
```

Comment:
Sadly, at the moment the various versions this page cares about outside of the `versions` list are defined in the template, not the view, so this would have been a refactor too far IMHO. The template tag was already there so this was simpler to keep.

In some places we have that logic encapsulated in models, with a custom transformer that sets a "private" property for a whole queryset and then some method that either returns the hidden property or re-computes the value if the transformer was not used. That would be my preference for code that we definitely need to re-use, but this case felt fairly specific...

Reply:
Makes sense